lib/board.rb

```diff
@@ -0,0 +1,56 @@
+require 'pry'
+
+class Board
+  attr_accessor :cells, :player
+
+  def initialize
+    @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+  end
+
+  def display
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+    puts " "
+    puts " "
+  end
+
+  def position(input)
+    index = input.to_i - 1
+    self.cells.fetch(index)
+  end
+
+  # def input_to_index(input)
+  #   index = input.to_i - 1
+  # end
+
+  def taken?(input)
+    index = input.to_i - 1
+    !(self.cells[index] == " ")
+  end
+
+  def reset!
+    @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
```

Comment:
Another way to instantiate an Array with `i` values is `Array.new(9, " ")` (see [Array.new](http://ruby-doc.org/core-2.2.0/Array.html#method-c-new)), where `9` is the number of elements in the array and `" "` is the default value for the elements. It's more concise and will prevent errors from repeatedly typing `" "`.

Reply:
Ah yes that is so much better